go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/coinbase/rosetta-sdk-go v0.8.2
+	github.com/ethereum-optimism/optimism/op-bindings v0.10.14
```

Comment:
We cannot add this dependency for all non op chains

Reply:
remove it and use expose as interface instead